indico/modules/core/controllers.py

```diff
@@ -283,3 +285,22 @@ def _process(self):
         self.user.reset_signing_secret()
         flash(_('All your token-based links have been invalidated'), 'success')
         return redirect(url_for('api.user_profile'))
+
+
+class RHConfig(RH):
+    def _process(self):
+        tos_url = legal_settings.get('tos_url')
+        tos_html = sanitize_html(legal_settings.get('tos')) or None
```

Comment:
Why `or None`?

Reply:
it may be an empty string, but we want explicit none on the client side

Reply:
Yeah, but I don't think we explicitly check for `None`. Empty string will anyway evaluate to false:

```javascript
{hasTos && tosHtml && (
    <Modal
    open={tosModal}
    closeIcon
    onClose={() => setTosModal(false)}
    trigger={
        <span>
        <a onClick={() => setTosModal(true)}>
            <Translate>Terms and Conditions</Translate>
        </a>
        </span>
    }
    >
    <Modal.Header>
        <Translate>Terms and Conditions</Translate>
    </Modal.Header>
    <Modal.Content>
        <div dangerouslySetInnerHTML={{__html: tosHtml}} />
    </Modal.Content>
    </Modal>
)}
```

Reply:
Ah, I guess it is better to have `null` here to avoid rendering empty strings in the DOM.